website/common.qmd

```diff
@@ -3,12 +3,37 @@ title: "Common Issues"
 editor: visual
 ---
 
-# Issue 1
+# Missing `\value`-tags in .Rd-files
 
 ## Problem
 
+Every .Rd file should have an `\value`-tag stating what the output of the function is. Even if nothing is returned, the `\value`-tag is necessary for CRAN.
```

Comment:
Do you have an example of what the actual error would look like from the R CMD check?

Reply:
Or rather what is an example message back from CRAN to a package maintainer?

Reply:
I don't know if R CMD check throws an error there, to be honest.

But what CRAN will message is something along those lines:

Please add \value to .Rd files regarding exported methods and explain the functions results in the documentation. Please write about the structure of the output (class) and also what the output means. (If a function does not return a value, please document that too, e.g. \value{No return value, called for side effects} or similar)

Followed by a list of all .Rd-files where the tag is missing.

Reply:
So the automatic R CMD check results are fine (i.e., indicate neither a NOTE or ERROR) and the package is forwarded to a manual check where this issue is then pointed out with this message in the response e-mail sent to maintainers?

Reply:
I think this has become a new check recently. When `R CMD check --as-cran` (+ some environmental variables set on CRAN?) is used it will return a warning or error (Because Bioconductor that uses `R CMD check` doesn't warn on this, as far as I know). I tried to find the commit or the documentation of this but I couldn't.